torch/_tensor_str.py

```diff
@@ -266,6 +266,23 @@ def _str(self):
         if values.numel() == 0:
             values_str += ', size=' + str(tuple(values.shape))
         tensor_str = indices_prefix + indices_str + '),\n' + ' ' * indent + values_prefix + values_str + ')'
+    elif self.is_quantized:
+        suffixes.append('size=' + str(tuple(self.shape)))
+        if not has_default_dtype:
```

Comment:
can you qtensor ever be the default dtype?  (It shouldn't be, but I don't know if it's checked).

Reply:
probably not